include/rgl/api/core.h

```diff
@@ -870,6 +886,17 @@ RGL_API rgl_status_t rgl_node_points_radar_track_objects(rgl_node_t* node, float
                                                          float object_radial_speed_threshold, float max_matching_distance,
                                                          float max_prediction_time_frame, float movement_sensitivity);
 
+/**
+ * Modifies RadarTrackObjectsNode to set entity ids to radar object classes mapping.
+ * This is necessary to call for RadarTrackObjectsNode to classify tracked objects correctly. If not set, objects will be classified as RGL_RADAR_CLASS_UNKNOWN by default.
```

Comment:
Did you mean "object classes" instead of "objects"? It is hard to tell whether an object will consist of multiple entities or not. A lot depends on user too (entities placement/configuration). On the other hand, objects can only belong to a single class.